prow/monitoring/BUILD.bazel

```diff
@@ -56,3 +56,22 @@ k8s_objects(
       ":servicemonitor_crd",
    ]
 )
+
+load("//def:secret.bzl", "k8s_secret")
+
+k8s_secret(
+   name = "alertmanager-prow",
+   cluster = "{STABLE_PROW_CLUSTER}",
+   namespace = "prow-monitoring",
+   data = {
+       "alertmanager.yaml": ":alertmanager.yaml",
+       "msg.tmpl": ":msg.tmpl",
```

Comment:
Why do we need to use a secret for these files instead of a configmap? They don't appear to contain secret data (especially given that you are checking them in).